conf/messages

```diff
@@ -229,7 +229,12 @@ hts.core.not-eligible.wtc.p-4-1=Find out what to do if you have checked the elig
 hts.core.not-eligible.wtc.p-4-link-text=think you&rsquo;re eligible
 hts.core.not-eligible.wtc.p-4-2=for a Help to Save account.
 
-hts.core.not-eligible.uc.p-1=This is because your household income - in your last monthly assessment period - was less than £542.88. Your Universal Credit payments are not considered to be income.
+hts.core.not-eligible.uc.p-1=This is because your household income - in your last monthly assessment period
+hts.core.not-eligible.uc.p-1b=This is because your household income in your last monthly assessment period
+hts.core.not-eligible.uc.p-1-2a=- was less than £
+hts.core.not-eligible.uc.p-1-2b= was not enough.
```

Comment:
is extra space after `=` necessary ?

Reply:
no it wasn't necessary, I have now removed it.